src/main/java/com/team4/testingsystem/entities/Question.java

```diff
@@ -16,7 +17,7 @@
 
 @Entity
 @Table(name = "question")
-public class Question {
+public class Question implements Serializable {
```

Comment:
do we really need it?

Reply:
Yes, we get the problem below without it
![image](https://user-images.githubusercontent.com/71790626/127976549-593fdd82-f4a7-436b-859f-c4b448373349.png)